.github/workflows/go.yml

```diff
@@ -46,6 +46,9 @@ jobs:
         # Use blst tag to allow go and bazel builds for blst.
         run: go build -v --tags=blst_enabled ./...
 
+      - name: Test static analyzers
```

Comment:
```suggestion
      - name: Run static analyzers
```

Reply:
Analyzer checks are already a part of the CI bazel build. This PR enables unit testing these analyzers during CI, so we're not running analyzers here.